editor/src/document/properties_panel_message.rs

```diff
@@ -0,0 +1,23 @@
+use crate::message_prelude::*;
+
+use serde::{Deserialize, Serialize};
+
+#[remain::sorted]
+#[impl_message(Message, DocumentMessage, PropertiesPanel)]
+#[derive(PartialEq, Clone, Debug, Serialize, Deserialize)]
+pub enum PropertiesPanelMessage {
+	ClearSelection,
+	MaybeDelete { path: Vec<LayerId> },
```

Comment:
Also `Delete` and then a path might be misinterpreted as deleting a layer, perhaps `clear` would be better?